requirements.txt

```diff
@@ -8,3 +8,5 @@ scipy
 joblib
 scikit-learn
 keras
+enum
+enum34
```

Comment:
Do we need both?

Reply:
Good point. Turns out we only need enum34.